### ISSUE TITLE
Offer to use a larger cert lineage rather than making a new subset lineage

### ISSUE THREAD

**Issue by pde** [certbot/certbot#1882]
If the user has a certs for a.com, b.com and c.com; and they ask for just a.com and b.com, we currently go ahead and make a new lineage.  Mightn't they want to use their existing cert/lineage instead?

**Comment by pde** [comment]
This is sort of more obvious once #1860 lands.

**Comment by bmw** [comment]
It seems we have the same behavior even with #1860.

**Comment by bmw** [comment]
Related #2128.

**Comment by schoen** [comment]
I think this could be considered superseded by #2128 although it's certainly still a valid problem. In particular, almost all situations that result in `-0001` weren't actually what the user intended, and are still a relatively frequent source of confusion. :frowning_face: So doing something useful to discourage the creation of such overlapping lineages will still be valuable to users.
